### PR TITLE
Fix client CI paths

### PR DESCRIPTION
**Pull request by PiddePannkauga**

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -1,6 +1,17 @@
 name: Node CI
 
-on: [push]
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - client/
```
Comment (Kalk88) [inline .github/workflows/nodejs.yml]:
```suggestion
    - client/*
```
If I understand the docs correctly. https://help.github.com/en/articles/workflow-syntax-for-github-actions

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+    
+  pull_request:
+    branches:
+    - master
+  paths:
+    - client/
```
Comment (Kalk88) [inline .github/workflows/nodejs.yml]:
```suggestion
    - client/*
```

Comment (PiddePannkauga) [reply .github/workflows/nodejs.yml]: I have same understanding after reviewing docs

File: .github/workflows/nodejs.yml [hunk 3]
```diff
@@ -1,6 +1,17 @@
 name: Node CI
 
-on: [push]
+on:
+  push:
+    branches:
+    - master
+    paths:
+      - client/*
```
Comment (Kalk88) [inline .github/workflows/nodejs.yml]:
```suggestion
    - 'client/**'
```

File: .github/workflows/nodejs.yml [hunk 4]
```diff
@@ -1,6 +1,17 @@
...
+
+  pull_request:
+    branches:
+    - master
+    paths:
+      - client/*
```
Comment (Kalk88) [inline .github/workflows/nodejs.yml]:
```suggestion
    - 'client/**'
```

**Pull request merged**